gamedata/descriptions/levels/privacy_complete.md

```diff
@@ -0,0 +1,3 @@
+Nothing in the ethereum blockchain is private. The keyword private is merely an artificial construct of the Solidity language. Web3's `getStorageAt(...)` can be used to read anything from storage. It can be tricky to read what you want though, since several optimization rules and techniques are used to compact the storage as much as possible.
+
+It can't get quite more complicated than what was exposed in this level. For more, check out this excellent article by "Darius": [How to read Ethereum contract storage](https://medium.com/aigang-network/how-to-read-ethereum-contract-storage-44252c8af925) 
```

Comment:
I would say "It can't get much more complicated than what was shown in this level", "quite" reads a little awkwardly.